src/XrdHttp/XrdHttpProtocol.cc

```diff
@@ -2601,6 +2604,12 @@ int XrdHttpProtocol::xexthandler(XrdOucStream &Config,
   }
 
   strcpy(path, val);
+
+  val = Config.GetWord();
+
+  if(val && !strcmp("notls",val)) {
```

Comment:
So, this introduces a problem. If the token is not 'notls' then it is ignored but according to the specification: https://xrootd.slac.stanford.edu/doc/dev56/xrd_config.htm#_Toc152596904 it could have been a parameter token to the plugin-in. Of course, the obvious solution is if it is 'notls' set the variable true and get another token. Otherwise, just pass it through (i.e. delete the subsequent 'get'). However, this creates another problem. If the token is notls then the exthandler will not see it and maybe notls is a valid exthandler parameter that now is technically missing. Unfortunately, the solution is to adopt the xroot style of of system parameters vs plug-in parameters which is somewhat more complicated. See my summary comments on how to do this.

Reply:
Thanks for your review. I've renamed the parameter as "+notls" and added it after the name of the external handler to load. That way, the path of the external handler can still be provided with extra parameters if necessary. https://github.com/xrootd/xrootd/pull/2253/commits/1b4aebd411bde17841c850c5598d8761ee1003a3

To be squashed once everything is OK before merging